addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2947,7 +2947,18 @@ var SnippetsMenu = Widget.extend({
         if (!ev.currentTarget.classList.contains('active')) {
             this._activateSnippet(false);
             this._mutex.exec(() => {
-                this._activateEmptyOptionsTab();
+                let $snippet = $();
```

Comment:
@bvr-odoo Maybe the use of "snippet" as name is confusing here

Reply:
I renamed it `$pageOptionsTarget`, I hope it's clearer.